chrome/extension/index.js

```diff
@@ -15,8 +16,8 @@ import '../../app/themes/index.global.scss';
 import BigNumber from 'bignumber.js';
 import { addCloseListener } from '../../app/utils/tabManager';
 
-// run MobX in strict mode
-useStrict(true);
+// run MobX in strict mode on non-prod
+configure({ enforceActions: environment.isMainnet() ? 'never' : 'always' });
```

Comment:
why this change? I think is easier to find problems if we are running in strict mode

Reply:
`always` is the same as `strict` (strict was renamed for who-knows what reason)